hotel-management-app-v1/src/main/java/com/fb/hotelmanagementappv1/models/Hotel.java

```diff
@@ -0,0 +1,45 @@
+package com.fb.hotelmanagementappv1.models;
+
+public class Hotel {
+    private int id;
+    private String name;
+    private String address;
+    private String telephoneNumber;
+    private int numberOfRooms;
```

Comment:
This about `HasA` relationship between Hotel and rooms. There is no point of having a hotel without rooms.